eng/scripts/Prepare-Release.ps1

```diff
@@ -119,6 +120,40 @@ catch
     $existing = @()
 }
 
+
+if (!$ReleaseDate)
+{
+    $currentDate = Get-Date
+    $thisMonthReleaseDate = Get-ReleaseDay((Get-Date -Day 1));
```

Comment:
I'm not sure what the standard is for release dates, but should we do something like try to find the first Tuesday of the month that isn't day 1?

Reply:
That's (almost) what we do https://github.com/pakrym/azure-sdk-for-net/blob/pakrym/na-contribute/eng/scripts/Prepare-Release.ps1#L78